sdk/maps/Azure.Maps.Common/samples/README.md

```diff
@@ -0,0 +1,25 @@
+eng\scripts\CodeChecks.ps1 -ServiceDirectory maps---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+name: Azure.Maps.Common samples for .NET
+description: Samples for the Azure.Maps.Common client library.
+---
+
+# Azure.Maps Samples
+
+## Import the namespaces
+
+To use common module, import `Azure.Maps` namespace:
+
+```C# Snippet:ImportCommonNamespace
+using Azure.Maps;
+```
+
+## Functionalities
```

Comment:
```suggestion
## Features
```

nit:  "Functionalities" gets flagged for grammar.    "Functionality" would work but there is no plural.